client/src/main/java/org/apache/falcon/entity/v0/EntityType.java

```diff
@@ -101,6 +101,16 @@ public boolean isSchedulable() {
         return ((this != EntityType.CLUSTER) && (this != EntityType.DATASOURCE));
     }
 
+    public static void assertSchedulable(String entityType){
```

Comment:
Can you use EntityType.getEnum inside this method? This method will be pretty much similar to AbstractSchedulableEntityManager.checkSchedulableEntity, except, the exception may be different.